storm-client/src/jvm/org/apache/storm/dependency/DependencyUploader.java

```diff
@@ -154,10 +160,15 @@ private boolean uploadDependencyToBlobStore(String key, File dependency)
             acls.add(new AccessControl(AccessControlType.OTHER,
                                        BlobStoreAclHandler.READ));
 
+            InputStream in = null;
             AtomicOutputStream blob = null;
             try {
+                in = Files.newInputStream(dependency.toPath());
```

Comment:
Please use try-with-resources as you did before, it is better and safer.

Reply:
Good idea! But, does two 'try's look pretty? also I want to keep the code style same with blob...
ok, I'll try the try-with-resources

Reply:
Thanks again for your advice. This is my first time to commit a PR,  I am learning how to do it formally. To save your time, when that being done, I'll notice you.